internal/sdk.go

```diff
@@ -68,6 +68,28 @@ func (d *SdkEnvs) ToEnvs() *env.Envs {
 	return envs
 }
 
+type AllSdk struct {
```

Comment:
`AllSdk` struct is unnecessary. We can simplify it by directly defining `AllSdk` as a type of map:

```golang
type Sdks map[string]*Sdk
```

Reply:
Since AllSdk is utilized in numerous places, controlling the order of maps directly where they are used would result in a significant amount of redundant code

Reply:
Just change the definitions, nothing else.